src/Discord/Repository/Guild/CommandPermissionsRepository.php

```diff
@@ -45,4 +45,14 @@ class CommandPermissionsRepository extends AbstractRepository
      * {@inheritDoc}
      */
     protected $class = CommandPermissions::class;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function __construct(Discord $discord, array $vars = [])
+    {
+        $vars['application_id'] = $discord->application->id; // For the bot's Application Guild Commands
```

Comment:
I'm not sure of what the plan is for the PR but regardless of that, this comment is pretty meaningless 😄

Reply:
it's not written by me, copied from another dphp file :D